Holovibes/includes/state/fast_updates_types.hh

```diff
@@ -31,6 +31,7 @@ enum class IndicationType
     IMG_SOURCE,
     INPUT_FORMAT,
     OUTPUT_FORMAT,
+    TEMPERATURE,
```

Comment:
Not fan in adding it in the IndicationType since temperature will more fit in FpsType (since it's an integer). And then in the information worker you change it to a string.

If you do do i think you Can rename FpsType to IntType.